src/main/java/org/folio/consortia/client/PoliciesClient.java

```diff
@@ -0,0 +1,16 @@
+package org.folio.consortia.client;
+
+import com.bettercloud.vault.json.JsonObject;
+import org.springframework.cloud.openfeign.FeignClient;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+
+import java.util.UUID;
+
+@FeignClient(name = "policies")
```

Comment:
can you please check if module descriptor has permissions for system user for newly added calls

Reply:
added required permission to system-user